hack/format-all-files.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+hack_dir="$(dirname $0)"
+repo_dir="$(dirname $hack_dir)"
+
+find . -iname "*.go" ! -wholename "./vendor/*" -print0 | \
+  xargs -0 go run "${repo_dir}/hack/cmd/coalesce-imports" -w
+find . -iname "*.go" ! -wholename "./vendor/*" -print0 | \
+  xargs -0 goimports -w -local github.com/projectcalico/calico/
```

Comment:
This is going to need to change in enterprise no? Can we / should we leverage metadata.mk to handle this since we've already got conditional logic for the GIT_REMOTE?

Reply:
Ah yep, good catch

Reply:
No, I think enterprise has the same Go package for calico itself, it's only the API that thinks of itself as `tigera/api`

Reply:
Oh, there's the third-party directory that Rene pointed out, I'll just add that in here since it should do no harm